cmd/syncer/cmd/syncer.go

```diff
@@ -71,6 +70,7 @@ func Run(options *synceroptions.Options, ctx context.Context) error {
 		&syncer.SyncerConfig{
 			UpstreamConfig:   globalhubConfig,
 			DownstreamConfig: toConfig,
+			SyncerNamespace:  options.PodNamespace,
```

Comment:
this is not a good idea for long term, we can't limit the syncer namespace. This is related to the global hub namespace issue, we need to figure out a better solution.
for now, it's working.

Reply:
Yes, I use this namespace as the identity of the syncer. When we integrate standalone control plane into the global hub api server, there will be a better way to solve this problem.